source/framework/core/src/TRestHits.cxx

```diff
@@ -546,28 +546,165 @@ Double_t TRestHits::GetSigmaY() {
 
     return sigmaY = TMath::Sqrt(sigmaY2);
 }
-
-Double_t TRestHits::GetGaussSigmaX(Int_t readoutChannels, Int_t startChannel, Int_t endChannel, Double_t pitch) {
+/**
```

Comment:
Same comment as on the `detectorlib` PR. It is better to just remove it. We will always be able to come back to the commit and check what were the changes happening.